python/cucim/src/cucim/core/operations/morphology/_distance_transform.py

```diff
@@ -153,7 +153,7 @@ def distance_transform_edt(
     """
     scalar_sampling = None
     if sampling is not None:
-        unique_sampling = np.unique(np.atleast_1d(sampling))
+        unique_sampling = cp.unique(cp.atleast_1d(sampling))
```

Comment:
Glad to see we can use CuPy here. Was there some kind of change that made this possible?

Reply:
It was always using CuPy, it was just getting called indirectly before via [NEP-18](https://numpy.org/neps/nep-0018-array-function-protocol.html#nep-18-a-dispatch-mechanism-for-numpy-s-high-level-array-functions)! It was just an oversight not to do `np`->`cp` previously in this file.

e.g.
```Python
import cupy as cp
import numpy as np

x = cp.ones(256)
y = np.unique(x)
type(y)
```
returns `cupy.ndarray`

Reply:
Ah ok. Well if NEP 18 makes it easier to reuse code, that seems like a plus